docs/common-tasks/using-watches.md

```diff
@@ -4,11 +4,7 @@ Often we need `watches` running inside of our containers. This could be for webp
 main challenges with the NFS mounts used in Outrigger is that they do not forward filesystem notifications across the 
 NFS mount and into containers, so we need to facilitate that.
 
-There is a command `rig watch [options] <path>`. Running this on your host will watch for changes and rsync 
-notifications to files and directories under `<path>` into the Docker Machine VM (which will then provide filesystem 
-notifications into the container). These are the options for `rig watch`
+`rig` now supports unison based file syncing between the local filesystem and a volume shared with a container. This
+setup allows multi-directional file syncing as well as full support for all filesystem notifications.
 
-* `--machine <name>` Optional: Specify a machine to send events. It will default to our `dev` machine
-* `--ignorefile <file>` Optional: Specify a file that contains patterns for directories/files to ignore.  Put one 
-entry per line (blank lines and comments are allowed). If not specified it will look for a file named 
-`.rig-watch-ignore` in the working directory and all parent directories.
+See [Filesystem Sync](../project-setup/filesystem-sync.md) for details on how to setup Unison volumes.
```

Comment:
Since not everyone will upgrade to the new system/approach right away, rather than remove docs couldn't we add a big, yellow-boxed deprecation warning at the top of the page with the details of what to look at instead?